includes/razorpay-affordability-widget.php

```diff
@@ -505,17 +505,28 @@ function getAffordabilityWidgetSettings()
                  'id'                   => 'wc_settings_tab_demo_section_end'
             ),
         );
-    } 
+    }
 
     return apply_filters('wc_affordability_widget_settings', $settings);
 }
 
-function displayAffordabilityWidgetSettings() 
+function initAdminFunctions()
 {
-    woocommerce_admin_fields(getAffordabilityWidgetSettings()); 
+    if (defined('WC_ABSPATH')) {
+        include_once WC_ABSPATH . 'includes/admin/wc-admin-functions.php';
+    }
+}
+
+function displayAffordabilityWidgetSettings()
+{
+    if(function_exists('woocommerce_admin_fields') === false)
+    {
+        initAdminFunctions();
```

Comment:
@rohitcbr as suggested by ram can we include logic here instead of creating initAdminFunctions()